app/src/main/java/co/nimblehq/compose/ecommerce/MainActivity.kt

```diff
@@ -14,6 +15,9 @@ import co.nimblehq.compose.ecommerce.ui.bottomnavigationbar.NavigationItem
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
+        WindowCompat.setDecorFitsSystemWindows(window, true)
```

Comment:
@hoangnguyen92dn Do we need to set this? If I'm not wrong, it will be `true` by default 🤔

Reply:
@Wadeewee Nooo 😄 It's required to pass true or false, no default value.

Reply:
> @Wadeewee Nooo 😄 It's required to pass true or false, no default value.

FYI: @hoangnguyen92dn I mean, why do we need to set `WindowCompat.setDecorFitsSystemWindows(window, true)` or what the purpose of putting it because I didn't see the change with/without this command ☝️  when I built the app 🙏

Reply:
@Wadeewee Oh...You're right, I added this to suppose display the full screen with status bar and navigation bar insets. I will get rid of this.